bpfman/src/oci_utils/image_manager.rs

```diff
@@ -93,9 +97,11 @@ impl ImageManager {
         // here: https://github.com/krustlet/oci-distribution/blob/main/src/reference.rs#L58
         let image: Reference = image_url.parse().map_err(ImageError::InvalidImageUrl)?;
 
-        self.cosign_verifier
-            .verify(image_url, username.as_deref(), password.as_deref())
-            .await?;
+        if let Some(cosign_verifier) = &mut self.cosign_verifier {
+            cosign_verifier
+                .verify(image_url, username.as_deref(), password.as_deref())
+                .await?;
+        }
```

Comment:
Add and else and a info!() log indicating Cosign is being skipped.

Reply:
Good point.  Done!